source/content/guides/object-cache-pro/02-installing-configuring.md

```diff
@@ -0,0 +1,238 @@
+
+In `config/application.php`, somewhere above the `Config::apply()` line at the bottom of the file, you'll want to add the following:
+
+```php
+/**
+ * Object Cache Pro config
+ */
+Config::define( 'WP_REDIS_CONFIG', [
+    'token' => '<LICENSE-TOKEN>',
+] );
+```
+
+I like to put this directly under the `WP_DEBUG` rules so it looks like this:
+
+```php
+/**
+ * Debugging Settings
+ */
+Config::define('WP_DEBUG_DISPLAY', false);
+Config::define('WP_DEBUG_LOG', false);
+Config::define('SCRIPT_DEBUG', false);
+ini_set('display_errors', '0');
+
+/**
+ * Object Cache Pro config
+ */ 
+Config::define( 'WP_REDIS_CONFIG', [
+    'token' => '<LICENSE-TOKEN>',
+] );
+```
+
+### Configuring Object Cache Pro
+There are a few other configuration options that must be added to your configuration file before Object Cache Pro will work. The instructions below provide guidance for both regular WordPress sites on Pantheon, and WordPress (Composer Managed)-based sites. Once the configuration options are added, there are still a couple steps before Object Cache Pro is fully ready-to-go.
+
+#### 1. Add Object Cache Pro configuration options
+
+The full, recommended contents of the `WP_REDIS_CONFIG` constant are as follows:
+
+```php
+	'token' => '<LICENSE-TOKEN>',
+	'host' => $_SERVER['CACHE_HOST'] ?? '127.0.0.1',
+	'port' => $_SERVER['CACHE_PORT'] ?? 6379,
+	'database' => $_SERVER['CACHE_DB'] ?? 0,
+	'password' => $_SERVER['CACHE_PASSWORD'] ?? null,
+	'maxttl' => 86400 * 7,
+	'timeout' => 1.0,
+	'read_timeout' => 1.0,
+	'debug' => false,
+	'analytics' => [
+		'enabled' => true,
+		'persist' => true,
+		'retention' => 3600, // 1 hour
+		'footnote' => true,
+	],
+```
+
+This should be added after `Config::define( 'WP_REDIS_CONFIG', [` in `config/application.php` for **WordPress (Composer Managed)**, or  `define( 'WP_REDIS_CONFIG', [` in `wp-config.php` for a **standard WordPress** site.
+
+There are more [configuration options](https://objectcache.pro/docs/configuration-options) that can be added here as well. We recommend using the above settings which are optimized for Pantheon if you don't know specifically what each option does.
+
+#### 2. Activate the plugin and enable Redis in the plugin
+
+Before attempting to activate the plugin, make sure you `git push` your changes up to your repository.
+
+You can activate the Object Cache Pro plugin from the admin, locally with Lando or via Terminus. In the WordPress admin, navigate to the Plugins page and activate Object Cache Pro. Then go to the Object Cache Pro settings(?) page, ensure that you are in an environment with file write permissions (either SFTP mode, if activating on your Pantheon Dev environment or in a local development environment), and click (?) the Enable (?) button. This will create the `object-cache.php` drop-in file. If you did this in your Pantheon Dev environment, be sure to commit the change in SFTP mode. If you did this locally, you'll want to commit the file to your repository.
```

Comment:
Do the question marks in this paragraph mean that we have to confirm the behavior? I know the path for OCP settings is `wp-admin/options-general.php?page=objectcache` and I think the others are correct based on [OCP docs](https://objectcache.pro/docs/installation#activation).

Reply:
Oh, good catch, yeah those were notes to myself for when I came back after testing! 🤣